.vscode/settings.json

```diff
@@ -5,5 +5,10 @@
     "source.fixAll.eslint":
     true
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#659bc4",
+    "titleBar.activeBackground": "#473d6a",
+    "titleBar.activeForeground": "#FDFAFE"
+  }
```

Comment:
Can you please remove these settings from here because this changes the settings of the workspace and it affects all developers who use VS Code and want to work on Wallet Selector?

To change only your own VS Code settings:
- CTRL + SHIFT + P 
- Then search the word "settings" 
- Choose Preferences: Open Settings (JSON).
- Write your own settings.

Reply:
A plugin made changes to the settings...I'll remove them.